assets/src/edit-story/utils/useAddScrollbarWidth.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useLayoutEffect, useState } from 'react';
+
+function useAddScrollbarWidth(refContainer) {
+  // State and callback ref necessary to recalculate the padding of the list
+  //  given the scrollbar width.
+  const [scrollbarWidth, setScrollbarWidth] = useState(0);
+  const refCallbackContainer = (element) => {
```

Comment:
I don't like the name of this method (also used in the two components). Yes, it is used as `ref={callback}` but it's not a reference - `refContainer` is.

Could we just name it `setContainer` or something like that?

Reply:
c300eb6